qemu/tests/cfg/device_option_check.cfg

```diff
@@ -41,6 +41,8 @@
                         force_drive_format = ide-drive
                     virtio_scsi:
                         force_drive_format = scsi-disk
+                    spapr_vscsi:
```

Comment:
Seems the 'special_drive_format' test is not necessary since the default 'drive_image' is 'scsi-hd'.

Reply:
Corrected as scsi-disk, thanks for reminding.